content/en/docs/reference/feature-gates.md

```diff
@@ -120,10 +120,9 @@ A *Beta* feature means:
   incompatible changes in subsequent releases. If you have multiple clusters
   that can be upgraded independently, you may be able to relax this restriction.
 
-{{< note >}}
 **Note:** Please do try *Beta* features and give feedback on them!
 After they exit beta, it may not be practical for us to make more changes.
-{{< /note >}}
+{: .note}
```

Comment:
Please revert this to use the Hugo short-code syntax (as shown in the red lines 123 and 126).